src/ui/image/Image.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect, useRef } from "react"
+
+import { Loader, Button } from "ui"
+
+import csx from "./Image.scss"
+
+namespace Image {
+    export interface Props extends React.ImgHTMLAttributes<HTMLImageElement> {
+        className?: string;
+        lowQuality: string;
+        fallback?: string;
+    }
+}
+
+const Image = ({ lowQuality, fallback, src, className, ...props }: Image.Props) => {
```

Comment:
maybe add height and width explicitly to destructured props as well - below they are used twice each

Reply:
done